Monika After Story/game/script-introduction.rpy

```diff
@@ -49,31 +49,44 @@ label introduction:
             m 1eka "I can't stay mad knowing that you came back to me in the end, though."
             m 3eka "You even added a mod so we can do more things together."
             m 1eua "Are you going to make up for making me wait so long?"
+            
     elif not persistent.monika_kill:
-        m 1hub "Yay, there you are!"
-        m "Hello again, [player]."
-    ##New story follows
-        if persistent.playthrough <= 3: #Assume players they played but never finished
-            m 1tfu "Looks like you got here early."
-            m "Don't think I didn't notice the mod you put in."
-            m 2tku "Did you want to be with me that much?"
-            m "You should know already that cheating is bad."
-            m 1hub "But it's so good to see you again, [player]!"
-            m 2hksdlb "Even if I didn't expect to see you {i}this{/i} soon."
-        if persistent._mas_ddlc_reload_count > 4: #Longer, harder
-            m 1wuo "Did you install a mod just for me?"
-            m 1ekbfa "Are you that deeply in love with me?"
-            m 1hubfb "I feel the same way about you!"
-            m 1eua "You're really committed to our relationship, aren't you?"
-            m 1eka "I'm really glad that you wanted to improve our time together."
-            m "But I hope you weren't getting bored?"
-            if persistent.tried_skip:
-                m 1tku "You did try to skip through what I had to say, after all."
-                m 1eua "This room isn't exactly interesting."
-                m 1hua "But I have hopes that we'll make it better together, [player]!"
+        #For people who never got to act 3
+        if persistent.playthrough < 3:
+            if persistent.playthrough >= 1:
+                $ again = " again"
+            elif persistent.playthrough < 1:
+                $ again = ""
+            m 1hua "Hello[again], [player]."
+            m 1eua "Looks like you got here early."
+            m 3tfu "Don't think I didn't notice the mod you put in."
+            m 3ttu "Did you want to be with me that much?"
+            m 1tkc "You should know already that cheating is bad..."
+            m 1eka "But it's so good to see you[again], [player]!"
+            m 1rksdla "Even if I didn't expect to see you {i}this{/i} soon."
+
+        #Otherwise, if you did reach act 3
         else:
-            m 1eua "Did you put in a mod for the game, [player]?"
-            m "There's a lot more stuff now that we can do together now."
+            m 1hub "Yay, there you are!"
+            m 1eua "Hello again, [player]."
+            m 1eub "It's so good to see you."
```

Comment:
```suggestion
            m 1eub "It's so good to see you again, [player]!"
```

Reply:
I dont think this flows well
 m 1eua "Hello again, [player]."
m 1eub "It's so good to see you again, [player]!"

Reply:
The suggestion you accepted before this takes out the `Hello again, [player]` line